### PR TITLE
Update api calls for sessions changes with that-sessions-api pr#65

### PR DESCRIPTION
**Pull request by brettski**
Updates website to work with graphql path change on create and update of sessions

### CODE REVIEW

File: components/Member/SessionEdit/Details.js [hunk 1]
```diff
@@ -120,6 +122,7 @@ const DetailForm = ({ sessionId }) => {
   const sessionIsWorkshop =
     session && session.type && session.type.indexOf('WORKSHOP') !== -1;
 
+  console.log(session);
```
Comment (kburnell) [inline components/Member/SessionEdit/Details.js:125]: does this need to be here?

Comment (csell5) [inline components/Member/SessionEdit/Details.js:125]: remove this..

File: components/Member/MySessions/Current.js [hunk 2]
```diff
@@ -98,10 +98,12 @@ const UPDATE_SESSION = gql`
   mutation updateSession($sessionId: ID!, $session: SessionUpdateInput!) {
     sessions {
       session(id: $sessionId) {
-        update(session: $session) {
-          id
-          title
-          status
+        update {
```
Comment (kburnell) [inline components/Member/MySessions/Current.js:101]: why is the double nested update necessary?

File: components/Session/Submit/Intro.js [hunk 3]
```diff
@@ -21,12 +21,14 @@ const dlog = debug('that:session:intro');
 const CREATE_SESSION = gql`
   mutation createSession($eventId: ID!, $session: SessionCreateInput!) {
     sessions {
-      create(eventId: $eventId, session: $session) {
-        id
-        type
-        title
-        category
-        status
+      create {
```
Comment (csell5) [inline components/Session/Submit/Intro.js:24]: the eventId is here now, not on the child create..

File: components/Member/SessionEdit/Details.js (hunk 1 above)
Comment (brettski) [reply components/Member/SessionEdit/Details.js:125]: whoopsie

Comment (brettski) [reply components/Member/SessionEdit/Details.js]: Fixed

File: components/Member/MySessions/Current.js (hunk 2 above)
Comment (brettski) [reply components/Member/MySessions/Current.js:101]: The update() mutation is now deprecated and here for backward compatibility.

File: components/Member/SessionEdit/Details.js (hunk 1 above)
Comment (brettski) [reply components/Member/SessionEdit/Details.js]: fixed

File: components/Session/Submit/Intro.js (hunk 3 above)
Comment (brettski) [reply components/Session/Submit/Intro.js:24]: @csell5 this is solved per our discussion, correct?

**Pull request merged**